### ISSUE TITLE
Feature Request: Resource for managing AWS Glue Jobs

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#3877]
Split feature request from #1416. For issues not immediately being worked on, please use :+1: upvotes on this original issue comment to help guage community interest.

### Terraform Version

```
terraform 0.10+
terraform-provider-aws (new feature)
```

### Affected Resource(s)

- `aws_glue_job`

### Terraform Configuration Files

```hcl
# New Resource - not implemented
resource "aws_glue_job" "example" {
  # ...
}
```

### Expected Behavior

Create, read, update, and delete support for AWS Glue jobs.

### Actual Behavior

Resource not implemented.

### References

* https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-jobs-job.html
* https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Glue.CreateJob

**Comment by FireballDWF** [comment]
If you produce a test build for me, I'm willing to beta test.  I had starting working earlier today on a terraform wrapping cloudformation to get this resource.

**Comment by bflad** [comment]
@FireballDWF thanks! I should it mostly finished up this afternoon or tomorrow. I'll reach out to you via email. 😄

**Comment by bflad** [comment]
PR submitted: #4028

**Comment by bflad** [comment]
The new `aws_glue_job` resource has been merged into master and will release with v1.15.0 of the AWS provider, likely mid next week.

**Comment by bflad** [comment]
This has been released in [version 1.15.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1150-april-18-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
